temba/channels/migrations/0129_auto_20211103_1310.py

```diff
@@ -63,9 +65,53 @@
 """
 
 
+def get_existing_messages_segments(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+    Msg = apps.get_model("msgs", "Msg")
+    ChannelLog = apps.get_model("channels", "ChannelLog")
+    ChannelCount = apps.get_model("channels", "ChannelCount")
+    msgs_to_update = dict()
+    channel_counts = list()
+    for log in (
+        ChannelLog.objects.using(db_alias)
+        .filter(
+            channel__channel_type__in=("T", "TW", "TMA", "TMS", "SW"),
+            is_error=False,
+            msg__isnull=False,
+        )
+        .exclude(Q(response__isnull=True) | Q(response=""))
+    ):
+        try:
```

Comment:
Could we please add a `print` command? So we can see the progress, I just ran this query on the production database and it has about 2k rows.

Reply:
Added requested changes.